src/main/java/core/basesyntax/service/impl/ParserImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.Parser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParserImpl implements Parser {
+    private static final String STRINGS_DELIMITER = " ";
+    private static final String FIELDS_DELIMITER = ",";
+    private static final int INDEX_OF_OPERATION = 0;
+    private static final int INDEX_OF_FRUIT = 1;
+    private static final int INDEX_OF_QUANTITY = 2;
+
+    @Override
+    public List<FruitTransaction> parseFruitTransactions(String dataFromFile) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        String[] stringsWithData = dataFromFile.split(STRINGS_DELIMITER);
+        for (String string : stringsWithData) {
```

Comment:
String string doesn't add any valuable information :) Variables' and constants' names are an opportunity to make your code more readable, don't squander it